lib/conffile.c

```diff
@@ -0,0 +1,176 @@
+#include <ctype.h>
```

Comment:
could we get some general comment descriptions of the functions in here, it took me a while to decode what was actually happening in a couple of them. Just a brief explanation of function and arguments.